mileage_tracker/views.py

```diff
@@ -0,0 +1,16 @@
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from mileage_tracker.forms import MileageForm
+from mileage_tracker.models import DistanceToWork
+
+# Create your views here.
+@login_required(login_url='login/')
+def set_commute(request):
+    if request.method == 'POST':
+        form = MileageForm(request.POST)
+        if form.is_valid():
+            miles = form.cleaned_data["miles"]
+            DistanceToWork.objects.create(user=request.user, miles=miles)
+        return render(request, "home.html")
```

Comment:
You probably don't want to render 2 different templates here.

I would think you want to:
- `GET` Show the user their currently stored DistanceToWork and present a form to create/update it.
- `POST` Process that form for creating/updating it.